client/settings/express-checkout/link-item.tsx

```diff
@@ -49,131 +48,104 @@ const LinkExpressCheckoutItem = (): React.ReactElement => {
 		availablePaymentMethodIds.includes( 'link' );
 	const isStripeLinkEnabled = enabledMethodIds.includes( 'link' );
 
+	if ( ! displayLinkPaymentMethod ) {
+		return null;
+	}
+
 	return (
-		<>
```

Comment:
I'd recommend to hide whitespaces while reviewing these changes: https://github.com/Automattic/woocommerce-payments/pull/10777/files?w=1

The main changes are:
- Moved the `displayLinkPaymentMethod` above, for an early return
- Removed the `HoverTooltip` component, replaced it with an `InlineNotice`

Reply:
🤔 This looks like an enhancement and irrelevant to the Linear issue here? May we move it to another PR.

Reply:
@htdat sorry, I missed this comment earlier - which part(s) of the changes do you consider an enhancement?